src/main/java/edu/ucla/library/prl/harvester/services/HarvestScheduleStoreService.java

```diff
@@ -34,9 +34,8 @@ public interface HarvestScheduleStoreService {
      * @return The service instance
      */
     static HarvestScheduleStoreService create(final Vertx aVertx, final JsonObject aConfig) {
-        // FIXME: this is incorrect, instantiate an implementing class instead
         // TODO: depending on implementation, consider returning Future<HarvestScheduleStoreService> instead
```

Comment:
Since a decision has been made (for now, at least) to not return a Future, I think this TODO can be removed too